tests_regression/test_rinoh.py

```diff
@@ -37,19 +37,36 @@ def collect_tests():
                if rst_path.stem == 'install_resources' else rst_path.stem)
```

Comment:
I checked this on master. It fails locally for me also, however the failure that I am seeing is not the one described in c540e33. 

The issue that I see is that the label for the internal cross-reference says "Chapter 2" rather than "example".

Reply:
> The issue that I see is that the label for the internal cross-reference says "Chapter 2" rather than "example".

Ah yes, this is because the default behavior has changed recently to label references by chapter/section number instead of by name. I'll need to update the reference PDF in this case.